generator/renderer.go

```diff
@@ -56,7 +58,26 @@ func (r *Renderer) CreateTrace(dir string, name string) error {
 
 	fmtsrc, err := format.Source(src.Bytes())
 	if err != nil {
-		return fmt.Errorf("cannot format source, %w", err)
+		switch err.(type) {
+		case scanner.ErrorList:
+			errorfile := "__tmp_error__.go"
+			fmt.Println("Bad Source:", dir, name)
+			fmt.Println(src.String())
+			fmt.Println("======")
+			for i, e := range err.(scanner.ErrorList) {
+				fmt.Printf("%d) cannot format source, %+v. %s\nadded error file in: %s", i, e, e.Error(), errorfile)
+			}
+			create, _ := os.Create(errorfile)
+			create.Write(src.Bytes())
+			create.Close()
+
+			if err != nil {
+				return err
+			}
+			return err
```

Comment:
Why do we need this error handling? 
It feels more complex than it should be.

Also it creates a file without checking for error, which means this code can lead to a panic situation. 

Can we keep all error reporting just printing to the stdout?

Reply:
yes, this was helping me to identify build issues, but I'll remove the file creation